fastlane/Fastfile

```diff
@@ -102,6 +102,27 @@ end
 
 desc "Installs all Certs and Profiles necessary for development and ad-hoc"
 lane :match_me do
+  run_match_me(readonly: true)
+end
+
+desc "Register new device, regenerates profiles, updates them remotely and locally"
+lane :register_new_device do
+  device_name = prompt(text: "Enter the device name: ")
+  device_udid = prompt(text: "Enter the device UDID: ")
+
+  register_device(
+    name: device_name,
+    udid: device_udid
+  )
+
+  run_match_me(readonly: false)
+end
+
+# Runs `match` for development and adhoc profiles of iOS apps/frameworks.
+# If `readonly: true`, it just donwloads the certs and profiles
+# If `readonly: false`, certs and profiles will be recreated, the new ones then
+# will be downloaded locally and automatically pushed to git repo.
+def run_match_me(readonly: true)
```

Comment:
The `readonly` can also be a `lane match` option.

Reply:
There was an idea to name this helper based on what it does (recreate_profiles...) but the name appeared to be long and the `fastlane` terminology was lost

Reply:
`readonly` being a `lane match_me` option with defaulting to `true` makes more sense